std/utf.d

```diff
@@ -1065,6 +1065,16 @@ if (isSomeChar!C)
 /// Whether or not to replace invalid UTF with $(LREF replacementDchar)
 alias UseReplacementDchar = Flag!"useReplacementDchar";
 
+// Reduce distinct instantiations of decodeImpl.
+private template CastForDecode(T)
```

Comment:
Please rename to something like `TypeForDecode`. It's named like it's a function, when its result is a type.

Reply:
Done.